lib/models/page.js

```diff
@@ -137,6 +137,10 @@ module.exports = function(crowi) {
     });
   };
 
+  pageSchema.methods.isRedirectOriginPage = function() {
+    return this.redirectTo != null;
```

Comment:
Please use `!== null` if `this.redirectTo` has only `null`.

Reply:
I am too embarrassed.
Thank you for pointing it out! I fixed :)